travis/dropbox.sh

```diff
@@ -16,4 +16,8 @@ else
     ~/dropbox_uploader.sh upload ./build/libs/*univ*.jar "pr/$TRAVIS_PULL_REQUEST"
 fi
 
+if ["$TRAVIS_BRANCH" == "develop"] || ["$TRAVIS_BRANCH" == "master"] || ["$TRAVIS_PULL_REQUEST" != "false"]; then
+    curl -s $UPLOADER_GENERATE_URL
```

Comment:
It's a variable and it downloads the newest jars (instead of every hour) so you see your newest build immediately. Don't want it public because of DDOS

Am 24.08.2016 01:22 schrieb Colton Schlosser <notifications@github.com>:

In travis/dropbox.sh:

> @@ -16,4 +16,8 @@ else > ~/dropbox_uploader.sh upload ./build/libs/*univ*.jar "pr/$TRAVIS_PULL_REQUEST" > fi > > +if ["$TRAVIS_BRANCH" == "develop"] || ["$TRAVIS_BRANCH" == "master"] || ["$TRAVIS_PULL_REQUEST" != "false"]; then > + curl -s $UPLOADER_GENERATE_URL 

Where is this url defined, and what is it?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.